packages/ckeditor5-paste-from-office/src/normalizers/mswordnormalizer.ts

```diff
@@ -22,13 +23,16 @@ const msWordMatch2 = /xmlns:o="urn:schemas-microsoft-com/i;
 export default class MSWordNormalizer implements Normalizer {
 	public readonly document: ViewDocument;
 
+	public readonly editor: Editor;
+
 	/**
 	 * Creates a new `MSWordNormalizer` instance.
 	 *
 	 * @param document View document.
 	 */
-	constructor( document: ViewDocument ) {
+	constructor( document: ViewDocument, edytor: Editor ) {
```

Comment:
It's `editor`, not `edytor`. This is a second typo I've noticed in this review. Maybe you could be a bit more careful when you write next time? :wink: